src/lib.rs

```diff
@@ -1,27 +1,30 @@
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
 
-// Expose parse_datetime
-pub mod parse_datetime;
-
-use chrono::{Duration, Local, NaiveDate, Utc};
-use regex::{Error as RegexError, Regex};
+use regex::Error as RegexError;
 use std::error::Error;
 use std::fmt::{self, Display};
 
+// Expose parse_datetime
+pub mod parse_relative_time;
+
+use chrono::{DateTime, FixedOffset, Local, LocalResult, NaiveDateTime, TimeZone};
+
+use crate::parse_relative_time::parse_relative_time;
```

Comment:
```suggestion
use parse_relative_time::parse_relative_time;
```
I think this should be enough, because we're already in the crate root. I might be wrong though.